kubernetes/keycloak.yaml

```diff
@@ -32,22 +52,127 @@ spec:
       containers:
         - name: keycloak
           image: quay.io/keycloak/keycloak:26.2.2
-          args: ["start-dev"]
+          args: ["start", "-Djgroups.bind.address=$(POD_IP)"]
```

Comment:
Might be better to put this with JAVA_OPTS_APPEND, since it also relates to jgroups.

Reply:
I found out the hard way that only the args will do the replacement `$(POD_IP)` - so I don't have a choice here. And this is the way the Operator does it.

Reply:
I was under the impression that you could use these expressions in environment variables also https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#using-environment-variables-inside-of-your-config

> And this is the way the Operator does it (probably for the same reason).

I think part of the motivation for that was not wanting to a "standard" environment variable that would cause us to need additional merging logic or release notes if something were already using that in the unsupported pod template.

Reply:
I figured out that re-ordering fixed it: First define `POD_ID`, then define `JAVA_OPTS_APPEND`. 
Applied to next commit.